modules/exploits/windows/http/nscp_authenticated_rce.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include ::Msf::Exploit::Remote::HttpClient
+  include ::Msf::Exploit::CmdStager
+  include ::Msf::Exploit::Powershell
+  prepend ::Msf::Exploit::Remote::AutoCheck
+  include ::Rex::Text
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'NSClient++ 0.5.2.35 - Authenticated Remote Code Execution',
+        'Description' => %q{
+          This module allows an attacker with knowledge of the admin password of NSClient++
+          to start a privilege shell.
+          For this module to work, both web interface of NSClient++ and `ExternalScripts` feature
+          should be enabled.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'kindredsec', # POC on www.exploit-db.com
+            'Yann Castel (yann.castel[at]orange.com)' # Metasploit module
+          ],
+        'References' =>
+          [
+            ['EDB', '48360']
+          ],
+        'Platform' => %w[windows],
+        'Arch' => [ARCH_X64],
+        'Targets' =>
+          [
+            [
+              'Windows',
+              {
+                'Arch' => [ARCH_X86, ARCH_X64],
+                'Type' => :windows_powershell
+              }
+            ]
+          ],
+        'Privileged' => false,
```

Comment:
The module documentation suggests that a privileged session is obtained. Does NSClient run as admin/system by default ? If so, this should be `true`. If not, then the module documentation is incorrect.

```suggestion
        'Privileged' => true,
```